assets/js/mixins/formatter.test.js

```diff
@@ -137,3 +137,21 @@ describe("fmtDuration", () => {
     config.global.mocks["$i18n"].locale = "de-DE";
   });
 });
+
+describe("getShortenedWeekdaysLabel", () => {
+  test("should format single days", () => {
+    expect(fmt.getShortenedWeekdaysLabel([0])).eq("Mo");
+    expect(fmt.getShortenedWeekdaysLabel([0, 2, 4, 6])).eq("Mo, Mi, Fr, So");
+    expect(fmt.getShortenedWeekdaysLabel([0])).eq("Mo");
+    expect(fmt.getShortenedWeekdaysLabel([3, 6])).eq("Do, So");
+  });
+  test("should format ranges", () => {
+    expect(fmt.getShortenedWeekdaysLabel([0, 1])).eq("Mo-Di");
+    expect(fmt.getShortenedWeekdaysLabel([0, 1, 2, 3, 4, 5, 6])).eq("Mo-So");
```

Comment:
Typografisch korrekt sind hier Leerzeichen und ein **En Dash** (` – `). Bei der Preisspanne in den Ladepunkt- oder Batterieeinstellungen haben wir das auch so gemacht.

Klingt wie eine Kleinigkeit, aber wir haben Nutzer (mich nicht ganz ausgenommen), denen diese Details sehr wichtig sind 😄

https://github.com/evcc-io/evcc/blob/master/assets/js/components/SmartCostLimit.vue#L286

![Bildschirmfoto 2024-09-23 um 10 00 03](https://github.com/user-attachments/assets/d3136e88-ca96-4a61-9490-f5ed0bf21530)

Reply:
Kann ich nachvollziehen, ist geändert. :D